doc/source/cookbook/downsampling_amr.py

```diff
@@ -8,19 +8,19 @@
 # If we ask for *all* of the AMR data, we get back field
 # values sampled at about 3.6 million AMR zones
 ad = ds.all_data()
-print(ad["gas", "density"].shape)
+print(ad[("gas", "density")].shape)
```

Comment:
I don't think this change is necessary, the accessor should work without parens as well (just checked it locally with a RAMSES dataset)

Reply:
This is strictly equivalent as in any case, the function `ad.__getitem__(self, keys)` will be called with `keys = ("gas", "density")`
```python
In [1]: class Toto:
   ...:     def __getitem__(self, keys):
   ...:         print(keys, type(keys))
   ...: 

In [2]: t = Toto()

In [3]: t[1, 2]
(1, 2) <class 'tuple'>

In [4]: t[(1, 2)]
(1, 2) <class 'tuple'>
```

Reply:
indeed. See my general comment bellow :)

Reply:
I think I've corrected all the cases of this.  But it seems like both are valid ways of specifying a tuple, so if I missed a few, it should be OK, I think.